xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/BittrexExchange.java

```diff
@@ -23,16 +31,16 @@
 
   private static final Object INIT_LOCK = new Object();
 
+  private static final Logger EXCHANGE_LOGGER = Logger.getLogger(BittrexExchange.class.getName());
```

Comment:
use this ocnvention:

`private static final Logger log = Logger.getLogger(BittrexExchange.class);`

Reply:
besides that we use slf4j API, not java.util.Logging

Reply:
Made the regarding changes in the latest commit.